deployment/charts/jenkins-master/values.yaml

```diff
@@ -4,38 +4,52 @@
 
 global:
   context: re
-  imageRegistry: "registry.hub.docker.com"
+  namespace: rpc-re
+  #imageRegistry: "registry.hub.docker.com"
+  imageRegistry: "docker-registry-default.devapps.rsi.rackspace.net"
+  imageRegistryInternal: "docker-registry.default.svc:5000"
 
 jenkins:
   name: "jenkins-master"
-  image: "dealako/re-jenkins"
-  imageVersion: "0.0.2"
+  image: "re-jenkins"
+  imageVersion: "0.0.3"
   pullPolicy: "Always"
   replicas: 1
   resources:
     requests:
       cpu: "100m"
-      memory: "2Gi"
+      memory: "4Gi"
     limits:
       cpu: "500m"
-      memory: "2Gi"
+      memory: "4Gi"
   network:
-    port: "8080"
-    protocol: "http"
-    targetPort: "8080"
-    status:
-      probeDelaySeconds: 600
-      probeTimeoutSeconds: 60
+    web:
+      port: "8080"
+      protocol: "tcp"
+      targetPort: "8080"
+      status:
+        probeDelaySeconds: 600
+        probeTimeoutSeconds: 60
+    slave:
+      port: "50000"
+      protocol: "tcp"
+      targetPort: "50000"
+      status:
+        probeDelaySeconds: 600
+        probeTimeoutSeconds: 60
   env:
     jenkinsArgs: "--prefix=/jenkins"
-    javaOpts: "-Dhudson.footerURL=\"http://www.rackspace.com\" -Djava.awt.headless=true -Djava.util.logging.config.file=\"/var/jenkins_home/logging.properties\""
+    javaOpts: "-Xmx2g -Xmx2g -Dhudson.footerURL=\"http://www.rackspace.com\" -Djava.awt.headless=true -Djava.util.logging.config.file=\"/var/jenkins_home/logging.properties\""
```

Comment:
one of these should probably be Xms

Reply:
Also why 2g when mem is 4g? To leave space for OS? I guess 3.5 would be ok?

Reply:
yeah, I meant -Xmx and -Xms - usually the same value and historically, we need to set the JVM to half the total memory available (per Elasticsearch rules which were a good benchmark).